File: travel_backend/src/main/java/com/example/travel_backend/dto/response/UserSettingResponseDto.java
package com.example.travel_backend.dto.response;

import lombok.Data;
import java.util.UUID;

@Data
public class UserSettingResponseDto {
    private UUID id;
    private Boolean pushReactions;
    private Boolean pushComments;
    private Boolean pushFollows;
    private Boolean pushReposts;
    private Boolean pushMentions;
    private Boolean pushAchievements;
    private String language;
    private String theme;
}

File: travel_frontend/app/src/main/java/com/example/vietnam_travel_itinerary_android/data/model/UserSettingResponseDto.kt
package com.example.vietnam_travel_itinerary_android.data.model

import java.util.UUID

data class UserSettingResponseDto(
    val id: UUID,
    val pushReactions: Boolean,
    val pushComments: Boolean,
    val pushFollows: Boolean,
    val pushReposts: Boolean,
    val pushMentions: Boolean,
    val pushAchievements: Boolean,
    val language: String,
    val theme: String
)